resources/zib/zib-Vaccination-event.xml

```diff
@@ -0,0 +1,158 @@
+        <comment value="Vaccination" />
+      </mapping>
+    </element>
+    <element id="Immunization.status">
+      <path value="Immunization.status" />
+      <comment value="This element has no functional counterpart in the zib and cannot be implicitly mapped to other concepts. Unless a more appropriate status is recorded, it is assumed that only _completed_ Immunization resources will be exchanged." />
+    </element>
+    <element id="Immunization.vaccineCode">
+      <path value="Immunization.vaccineCode" />
+      <short value="ProductCode" />
+      <definition value="The product code of the vaccine administered.&#xD;&#xA;There are several possible code systems for documenting the product code. If the vaccination data is registered based on an anamnesis, coding with the ATC code is preferred. In all cases it concerns those products that fall under ATC group J07 (vaccines)." />
+      <alias value="ProductCode" />
+      <binding>
+        <strength value="example" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.12--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-vaccination-v4.0-2020EN" />
+        <map value="NL-CM:11.1.2" />
+        <comment value="ProductCode" />
+      </mapping>
+    </element>
+    <element id="Immunization.occurrence[x]">
+      <path value="Immunization.occurrence[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Immunization.occurrence[x]:occurrenceDateTime">
+      <path value="Immunization.occurrence[x]" />
+      <sliceName value="occurrenceDateTime" />
+      <short value="VaccinationDate" />
+      <definition value="Date (and if possible time) that the vaccine was or will be administered. In the case of a planned re-vaccination, a vague date (month, year) is allowed (Clinical reminder). If a planned vaccination cannot be administered before a certain date, the date stated (vague or not) must indicate this." />
+      <alias value="VaccinatieDatum" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-vaccination-v4.0-2020EN" />
+        <map value="NL-CM:11.1.3" />
+        <comment value="VaccinationDate" />
+      </mapping>
+    </element>
+    <element id="Immunization.doseQuantity">
+      <path value="Immunization.doseQuantity" />
+      <short value="Dose" />
+      <definition value="The amount of product administered shown in milliliters. In most cases, the entire product is administered; in some cases, a described part of the product is administered." />
+      <alias value="Dosis" />
+      <mapping>
+        <identity value="zib-vaccination-v4.0-2020EN" />
+        <map value="NL-CM:11.1.4" />
+        <comment value="Dose" />
+      </mapping>
+    </element>

```

Comment:
Als de definition zegt dat het over mL gaat, kunnen we dan niet de unit (en system, etc.) slimmer maken?

Reply:
Done! Goeie.